css/style.css

```diff
@@ -0,0 +1,293 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+header {
+  min-width: 100%;
+  margin-top: 70px;
+}
+
+header > div {
+  background-color: #f72210;
+}
+
+.header-btn div {
+  float: right;
+  margin-bottom: 20px;
+}
```

Comment:
- [ ] You should use the bootstrap class `float-right` here instead of css classes